### PR TITLE
fix(dropdown):update border colour

### LINKED ISSUE #633: Dropdown component hover color
The hover for the dropdown component is not supposed to be blue as it is now.

Border:  pf-black-600 2px solid
Text: pf-black-900.

pf-dropdown-component

### PR DESCRIPTION
**Pull request by matthewcarleton**
Fixes #633 

This also removes the `.pf-m-action` from the kebab that is right aligned and adds the `pf-m-plain` class. @jgiardino please let me know if I missed something and that should have remained.

### CODE REVIEW

File: src/patternfly/components/Dropdown/styles.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
   --pf-c-dropdown__toggle--BorderRadius: var(--pf-global--BorderRadius--sm);
   --pf-c-dropdown__toggle--Color: var(--pf-global--Color--100);
   --pf-c-dropdown__toggle--hover--BorderWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-dropdown__toggle--hover--BorderColor: var(--pf-global--active-color--100);
+  --pf-c-dropdown__toggle--hover--BorderColor: var(--pf-global--BorderColor--dark);
```
Comment (michael-coker) [inline src/patternfly/components/Dropdown/styles.scss]: Why use `--pf-global--BorderColor--dark` here when the non-hover border color is `--pf-global--BorderColor`? Those colors are the same value, so it doesn't matter.

Although could we just remove the `border-color` variable and css, and just modify the `border-width` on hover?

Comment (matthewcarleton) [reply src/patternfly/components/Dropdown/styles.scss]: I've updated, but we need to keep the variable b/c it's used further down for the plain kebab hover. Although this exposes another issue of the plain hover when expanded vs the pf-m-action hover when expanded. I assume they should be the same.

File: src/patternfly/components/Dropdown/styles.scss [hunk 2]
```diff
@@ -82,15 +82,15 @@
   &:hover,
   &.pf-m-hover {
     &::before {
-      border-color: var(--pf-c-dropdown__toggle--hover--BorderColor);
       border-width: var(--pf-c-dropdown__toggle--hover--BorderWidth);
```
Comment (jgiardino) [inline src/patternfly/components/Dropdown/styles.scss]: When I compare the pf-m-plain variant of button to dropdown, the button never shows a border, but the dropdown shows a border on hover and expanded state. The expanded state makes sense, but are the hover states meant to be similar?

**Comment by jgiardino** [comment]
I'm not sure what the expected border is on hover for pf-m-plain buttons vs pf-m-plain dropdowns, and commented inline. But other than that, these updates look good to me!

File: src/patternfly/components/Dropdown/styles.scss (hunk 2 above)
Comment (matthewcarleton) [reply src/patternfly/components/Dropdown/styles.scss]: thanks Jenn. Ya originally the plain dropdown had a hover a blue border I changed it to the pf-black-600. I do think that the plain button and plain dropdown should align for hover. I'm going to check in with @mceledonia on this one.

**Pull request merged**